templates/repo/issue/view_content/comments.tmpl

```diff
@@ -317,110 +317,132 @@
 				{{else}}
-					<a{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>{{.Poster.GetDisplayName}}</a>
+				<a class="timeline-avatar"{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>
+					<img src="{{.Poster.RelAvatarLink}}">
+				</a>
 				{{end}}
+				<span class="badge {{if eq .Review.Type 1}}green
+				{{- else if eq .Review.Type 2}}grey
+				{{- else if eq .Review.Type 3}}red
+				{{- else}}grey{{end}}">{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}</span>
+				<span class="text grey">
+					{{if .OriginalAuthor }}
+						<span class="text black"><i class="fa {{MigrationIcon $.Repository.GetOriginalURLHostname}}" aria-hidden="true"></i> {{ .OriginalAuthor }}</span><span class="text grey"> {{if $.Repository.OriginalURL}}</span><span class="text migrate">({{$.i18n.Tr "repo.migrated_from" $.Repository.OriginalURL $.Repository.GetOriginalURLHostname | Safe }}){{end}}</span>
+					{{else}}
+						<a{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>{{.Poster.GetDisplayName}}</a>
+					{{end}}
 
-				{{if eq .Review.Type 1}}
-					{{$.i18n.Tr "repo.issues.review.approve" $createdStr | Safe}}
-				{{else if eq .Review.Type 2}}
-					{{$.i18n.Tr "repo.issues.review.comment" $createdStr | Safe}}
-				{{else if eq .Review.Type 3}}
-					{{$.i18n.Tr "repo.issues.review.reject" $createdStr | Safe}}
-				{{else}}
-					{{$.i18n.Tr "repo.issues.review.comment" $createdStr | Safe}}
-				{{end}}
-			</span>
+					{{if eq .Review.Type 1}}
+						{{$.i18n.Tr "repo.issues.review.approve" $createdStr | Safe}}
+					{{else if eq .Review.Type 2}}
+						{{$.i18n.Tr "repo.issues.review.comment" $createdStr | Safe}}
+					{{else if eq .Review.Type 3}}
+						{{$.i18n.Tr "repo.issues.review.reject" $createdStr | Safe}}
+					{{else}}
+						{{$.i18n.Tr "repo.issues.review.comment" $createdStr | Safe}}
+					{{end}}
+				</span>
+			</div>
 			{{if .Content}}
-				<div class="detail">
-					{{svg "octicon-quote" 16}}
-					<span class="text grey has-emoji">{{.Content}}</span>
+			<div class="timeline-item comment">
+				<div class="content">
+					<div class="ui top attached header arrow-top">
+						<span class="text grey">
+							{{if .OriginalAuthor }}
+								<span class="text black"><i class="fa {{MigrationIcon $.Repository.GetOriginalURLHostname}}" aria-hidden="true"></i> {{ .OriginalAuthor }}</span><span class="text grey"> {{if $.Repository.OriginalURL}}</span><span class="text migrate">({{$.i18n.Tr "repo.migrated_from" $.Repository.OriginalURL $.Repository.GetOriginalURLHostname | Safe }}){{end}}</span>
+							{{else}}
+								<a{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>{{.Poster.GetDisplayName}}</a>
+							{{end}}
+
+							{{$.i18n.Tr "repo.issues.review.left_comment" | Safe}}
+						</span>
+					</div>
+					<div class="ui attached segment">
+						<div class="detail">
+							<span class="text grey has-emoji">{{.Content}}</span>
```

Comment:
This should be black, else all review comments are grey

Reply:
there is much more such things, like 4 different styles for author's name

Reply:
I don't think I understand, this styling is just for the content not the author name

Reply:
it's something that is changed by this PR? it was just copied, as you can see here https://github.com/go-gitea/gitea/pull/10972/files/bf084000712e45eb4c1b92bb5eac213634f8f57c#diff-7b9c057e968c3c88205ca2b63f0bfbfbL351

I was just pointing that there are inconsistencies all over the places...

Reply:
It was grey before, but it was also displayed in the timeline before and not as its own comment. I think displaying it as a comment it should have black text like other comments.

Since this PR is about making lots of changes to the look it is ok to change things from how they were before if you think they will look better another way. If you think something seems bad or wrong feel free to make/suggest a change

Reply:
I'm afk for a few days, if you can fix it, simply, by removing `gray` class pls push to this branch you should still have rights

I'll try to fix some more thinks in further PRs